tests/fixtures/application_fixtures.py

```diff
@@ -2807,6 +2807,7 @@ def tcf_system(db: Session) -> System:
             "legal_basis_for_processing": "Consent",
             "egress": None,
             "ingress": None,
+            "retention_period": "3-5 days",
```

Comment:
nit: i know we want to accommodate "arbitrary" string values for this property, i'm just a bit dubious of putting this value in a fixture that's meant to be representative of a TCF/GVL system. the GVL systems should really only have integer-like values (e.g. `"397"`) on this field. i'm a bit torn though because i know we want to have test coverage for values like `"3-5 days"` since that's possible for non-GVL systems. i just get worried about establishing a fixture here that's specifically different from the data we're expecting upstream that it's meant to represent.

would it be too much to create another fixture perhaps that's a non-GVL system? it may not be bad to have test coverage for that generally on the BE...although to be fair, i'm not 100% sure it's gonna show up in the overlay at all _yet_ 😄

Reply:
based on the convo with Rachel today, I think I'll change these to just a number since it sounds like that's pretty much what we'll expect for the future